.github/workflows/gradle.yml

```diff
@@ -487,3 +487,116 @@ jobs:
           github-token: ${{ secrets.SELF_RUNNER_GITHUB_ACCESS_TOKEN }}
           label: ${{ needs.start-kube-acceptance-test-runner.outputs.label }}
           ec2-instance-id: ${{ needs.start-kube-acceptance-test-runner.outputs.ec2-instance-id }}
+
+  start-gke-kube-acceptance-test-runner:
```

Comment:
something that will go away once we move to on demand testing - running this on every build might overload the tiny GKE cluster we have now.

Reply:
Oh am going to add the master and cron condition in here. I have not added it cause I wanted to run this on the CI and see how it goes